kubeshop/testkube-dashboard
# fix: override escape-carriage package version for better logs performance

## Changes

- override `escape-carriage` package version for better logs performance

## Fixes

- https://github.com/kubeshop/testkube/issues/3522

## How to test it

- connect with `https://demo.testkube.io/results/v1` API
- use `/tests/executions/maven-example/execution/64217ab34afc10c4911b05c6` URL
- it should work fine

## Checklist
